README.md

```diff
@@ -53,13 +53,23 @@ We are iterating and looking for feedback and collaboration, so please [**let us
   <img src="https://img.shields.io/pub/v/amplify_api.svg">
 </a>
 
+## Category / Platform Support (Developer Preview)
```

Comment:
Can we add one for stable as well? Even though it's a bit redundant, it might be good to highlight for comparison with this.

Reply:
@dnys1 Do you mean an equivalent table in the `stable` branch, or two tables in `next`?

Reply:
I think we need this table and a matching one for `stable` in the top-level `README` on `main` branch and in the package READMEs on both branches.